Block/Info.php

```diff
@@ -33,14 +33,12 @@ protected function _prepareSpecificInformation($transport = null)
         $boltProcessor = $info->getAdditionalInformation('processor');
```

Comment:
@vitaliyreznikov For for adyen, alliance, braintree, nmi, and stripe, what's the value of $info->getAdditionalInformation('processor'); ?

Reply:
@dumega For now we have following processor's type:

adyen_gateway
affirm	
afterpay	
alliance_data
amazon_pay	
bolt	
braintree
nmi
paypal	
stripe
vantiv

Reply:
But for Paypal and afterpay , we do not have cc type and cc last 4.

Reply:
Yes, but we still can use getter and receive empty value.
I don't want to have a full list of supported processors here because we need to update it each time we support new payment processor

Reply:
Make sense. Then pleae remove `$boltProcessor = $info->getAdditionalInformation('processor');` as well, cause now it is useless in this function.

Reply:
Good catch @dumega ! Done.